pfcpiface/up4.go

```diff
@@ -73,6 +73,25 @@ type meter struct {
 	downlinkCellID uint32
 }
 
+// tnlPeerReference <F-SEIDs (UE sessions); FAR-ID> pair that
+// uniquely identifies tunnel peer among different FAR IEs of the same UE session.
+type tnlPeerReference struct {
+	fseid uint64
+	farID uint32
+}
+
+type tunnelPeer struct {
+	id uint8
+	// usedBy keeps track of <F-SEIDs (UE sessions); FAR-ID> pairs using this tunnel peer.
+	// It's implemented as Set to avoid duplicated F-SEIDs.
```

Comment:
I suggest removing this comment, it sounds confusing now that we store fully qualified FAR IDs.  According to the spec, there should not be two FARs with the same ID in the same session.